bin/wmagent-component-standalone

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+"""
+wmagent-component-standalone
+
+Utility script for running a WMAgent component standalone.
+"""
+
+import os
+import sys
+import logging
+import importlib
+
+from argparse import ArgumentParser
+from WMCore.Configuration import loadConfigurationFile
+from WMCore.ResourceControl.ResourceControl import ResourceControl
```

Comment:
I wonder why these 2 explicit imports here (ResourceControl and CRIC)? Aren't we supposed to get them as a dependency of the component that we are importing?

Reply:
Well, since this was thought of and designed as auxiliary debugging script, my idea was  to put an explicit set of imports of commonly used modules, regardless of the component we  run in standalone mode.